examples/basic/async_data_transfer_client.py

```diff
@@ -80,19 +82,23 @@ async def main(
     )
     await client.start()
 
-    ########################################
-    # Create a AsyncDataTransferApi instance
-    # ======================================
+########################################
+# Create a AsyncDataTransferApi instance
+# ======================================
 
     api = AsyncDataTransferApi(client)
     await api.status(wait=True)
 
-    log.info("Query storages ...")
+########################################
+# Get available storages
+# ======================
     storages = await api.storages()
     storage_names = [f"{storage['name']}({storage['type']})" for storage in storages]
     log.info(f"Available storages: {storage_names}")
 
-    log.info("Creating a directory ...")
+########################################################################
+# Create a directory, copy files, list files, get metadata, remove files

```

Comment:
Can the section title be shortened to something like "Set up project"? The current title is quite long!